DSA/Cpp/Recursion_and_Backtracking/TowerOfHanoi.cpp

```diff
@@ -0,0 +1,17 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+void TOH(int n, char a, char b, char c)
+{
+    if (n == 0)
+        return;
+    TOH(n - 1, a, c, b);
+    cout << n << " [ " << a << " -> " << b << " ]" << endl;
+    TOH(n - 1, c, b, a);
+}
+
+int main()
+{
+    TOH(3, 'A', 'B', 'C');
+    return 0;
+}
```

Comment:
Do not add a code that you didn't mention in the issue or the PR in the first place.